src/components/Header/Header.jsx

```diff
@@ -1,9 +1,39 @@
-import './header.scss'
+import React from "react";
+import "./header.scss";
+import burgerOpen from "../../assets/header/burger-menu.svg";
 
 const Header = () => {
   return (
-    <div>Header</div>
-  )
-}
+    <div className="header-container">
+      <a className="header-container-logo" href="/index.html">
+        <img
```

Comment:
please, use BEM for classnames) 
you are so close, but the correct form must be header-container__logo, etc.